week2-section2/exercise2/PlayList.cpp

```diff
@@ -0,0 +1,61 @@
+#include <iostream>
+#include "Song.h"
+#include"PlayList.h"
```

Comment:
Since you are including "Playlist.h" and "Playlist.h" already includes <iostream> and "Song.h", there is no need to include them here as well.